Tasks/google-play-promote/task.json

```diff
@@ -23,27 +23,27 @@
         {
             "name": "authType",
             "type": "pickList",
-            "label": "Authentication Method",
+            "label": "Authentication method",
             "defaultValue": "ServiceEndpoint",
             "helpMarkDown": "",
             "options": {
                 "JsonFile": "JSON Auth File",
-                "ServiceEndpoint": "Service Endpoint"
+                "ServiceEndpoint": "Service connection"
             }
         },
         {
             "name": "serviceEndpoint",
```

Comment:
Can you please add an alias for the `serviceEndpoint` input and name it `serviceConnection`?